From 405bf4198447f9c9cfee901f9c07bf6479c182f9 Mon Sep 17 00:00:00 2001
From: txtsd <code@ihavea.quest>
Date: Fri, 21 Nov 2025 16:30:10 +0530
Subject: [PATCH] Patch for outdated python-isocodes

Signed-off-by: txtsd <code@ihavea.quest>
---
 comicapi/utils.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/comicapi/utils.py b/comicapi/utils.py
index 9e7784b..0e068fc 100644
--- a/comicapi/utils.py
+++ b/comicapi/utils.py
@@ -581,7 +581,7 @@ def languages() -> dict[str | None, str | None]:
     if not _languages:
         import isocodes
 
-        for alpha_2, lng in isocodes.extended_languages._sorted_by_index(index="alpha_2"):
+        for alpha_2, lng in isocodes.extendend_languages._sorted_by_index(index="alpha_2"):
             _languages[alpha_2] = lng["name"]
     return _languages.copy()
 
@@ -602,7 +602,7 @@ def get_language_iso(string: str | None) -> str | None:
 
     found = None
 
-    for lng in isocodes.extended_languages.items:
+    for lng in isocodes.extendend_languages.items:
         for x in ("alpha_2", "alpha_3", "bibliographic", "common_name", "name"):
             if x in lng and lng[x].casefold() == lang:
                 found = lng
-- 
2.51.2

